websites/G/Google/presence.ts

```diff
@@ -9,9 +9,10 @@ const presence = new Presence({
 
 presence.on("UpdateData", async () => {
 	const presenceData: PresenceData = {
-		largeImageKey: "lg",
-		startTimestamp: browsingTimestamp
-	};
+			largeImageKey: "lg",
+			startTimestamp: browsingTimestamp
+		},
+		privacy = await presence.getSetting<boolean>("privacy");
 
 	if ((homepageInput && homepageImage) || !document.location.pathname) {
 		presenceData.state = "Home";
```

Comment:
no clue why this is in the state but I think you can mention the user is in home with privacy on, so this should go to the details. Also do me a favour and remove the line right below this one as it's unnecessary

Reply:
I'll do it ASAP